IchigoJam_T

T-Sa-sa-ki opened the pull request:
I have made a new part of "IchigoJam T" now.
IchigoJam T is an one-board PC for education.

This issue was discussed in Fritzing Forum.
http://forum.fritzing.org/t/ichigojam-t-as-a-kids-pc/4997/15

el-j commented:
thank you for the new part. could you please add the view-name at the end of the svg-filename.
like:

```
partname_breadboard.svg
partname_schematic.svg
partname_pcb.svg
partname_icon.svg
```

T-Sa-sa-ki commented:
Thank you for your replying.
I have just added the view-name at the end of the svg-filename and replaced now.
as follows:
> IchigoJam_T_breadboard.svg
> IchigoJam_T_schematic.svg
> IchigoJam_T_pcb.svg
> IchigoJam_T_icon.svg
